tests/parsers/test_utils.py

```diff
@@ -71,6 +79,21 @@ def test_single_entry(self, monkeypatch, tmpdir, entries):
             TOCEntry(name="foo", type="Method", anchor="foo")
         ] == parser._patched_entries
 
+    def test_single_entry_url_format(
+        self, monkeypatch, tmpdir, entries_url_format
+    ):
+        foo = tmpdir.mkdir("foo")
+        foo.join("foo bar.html").write("docs!")
+        parser = FakeParser(doc_path=str(foo))
+        toc = patch_anchors(parser, show_progressbar=False)
+        for e in entries_url_format:
+            print(e)
+            toc.send(e)
+        toc.close()
+        assert [
+            TOCEntry(name="foo", type="Method", anchor="foo")
```

Comment:
wait, doesn't this mean that you've lost a TOCEntry somewhere?

Reply:
I'm lost on writing a test. I though I could just add a new entry to the `entries` fixture, copy the `test_single_entry` test in `test_utils.py` and change it so it checks all the entries have been correctly parsed. But I don't understand what is happenning in here.

Reply:
No worries, I’ll have a look after Xmas. Happy holidays and thanks for your work so far!

Reply:
Thanks, and happy holidays to you too!